docker/Dockerfile-ubuntu-20.04

```diff
@@ -18,8 +18,9 @@ RUN cd /root && git clone --recurse-submodules -b v1.58.0 https://github.com/grp
     && cd grpc/ && mkdir build && cd build \
     && cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF -DCMAKE_CXX_STANDARD=17 ../ \
     && make -j4 \
-    && make install
-    && cd grpc/ && mkdir -p cmake/build && cd cmake/build && cmake -DgRPC_BUILD_TESTS=ON ../.. && make grpc_cli
+    && make install \
+    && cd grpc/ && mkdir -p cmake/build && cd cmake/build \
+    && cmake -DgRPC_BUILD_TESTS=ON ../.. && make grpc_cli
```

Comment:
I believe we also need to end line 23 with a backslash, right?

Reply:
Yep, you're right!